### DISCUSSION TITLE
Im making changes in 2nd and 10th user, applies only to admin user.

### DISCUSSION THREAD

**Discussion by kangarooo** [Sophia-Community/SophiApp#17] [General]
Im making changes in 2nd user, applies only to admin user.
Nothing changes in user im changing changes.
Except selecting dark app mode, this app became dark but nothing in user, only all was applied in admin that i dont use.

**Comment by farag2** [comment]
The app isn't finished yet.

**Comment by farag2** [comment]
When you open the app, do you see thr UAC prompt?

**Reply by kangarooo** [reply]
Si sinjor, yes. It asks only for Admin password. Users dont have passwords. Doesnt allow to choose to do User.

**Reply by farag2** [reply]
If you're required to type the adnin password, it means, the current user isn't an admin. Am I right?

**Reply by kangarooo** [reply]
Yes, admin password it asks  ̶b̶u̶t̶ ̶N̶o̶,̶ ̶ Yes current user is (not admin) User10. Opening asks for Password. User10 doesnt have password. Entering admin password works app in User10. Nothing is applied to User10. Accidentally opened Admin and checked that all was applied there where didnt needed to apply in there. It made changes in Admin user from User10.

**Reply by farag2** [reply]
I hardly could translate it. :) Does your admin account have a password?

**Reply by kangarooo** [reply]
Yes, admin has password. Im was user2 and user10 without passwords. What language you need? I can write in it. You asked.
If you're required to type the adnin password, it means, the current user isn't an admin. 
Yes, admin password it asks but No, current user is User10.
Now im fired from work for rocking up admin account.
Can in communist language ах да, невидел што написал isnt, читал думал што is
спрашивал Админ password потому што аццоунт User2 User10 не админ - edited now that  User2 and User 10 is not admin
шас меня уволили

**Reply by farag2** [reply]
> Yes, admin password it asks but No, current user is User10.

That's I was talking about. The app won't work if the current admin isn't an admin.
